suites/pacific/rados/tier-3_rados_ssd-tests.yaml

```diff
@@ -124,9 +155,14 @@ tests:
                   service_id: hdd_nodes
                   placement:
                     nodes:
-                      - node1
-                      - node2
-                      - node3
+                      - node4
+                      - node5
+                      - node6
+                      - node7
+                      - node8
+                      - node9
+                      - node12
+                      - node13
                   spec:
```

Comment:
Purpose of the test is to add OSDs on SSDs and HDDs, but spec file is designed to deploy OSDs on rotational devices only.
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-9264

Reply:
Looks good. The deployment happens twice, once with HDDs and once with SSDs.

Reply:
@pdhiran where? The other deployment is with only SSD drives and has a different Polarion ID.
If we decide to proceed with deployment of OSDs on HDDs and SSDs with different spec files, we should modify Polarion test case to reflect the same.

Reply:
Yes, Sorry, my bad. The polarion IDs are different. Let's update the polarion tests.

What i meant was the suite file covered the workflow for deployment of both HDD and SSD OSDs.